src/palette/internal/palettelayout.cpp

```diff
@@ -359,8 +359,8 @@ void PaletteLayout::layout(Ambitus* item, const Context& ctx)
 
 void PaletteLayout::layout(Arpeggio* item, const Context& ctx)
 {
-    engraving::layout::v0::LayoutContext ctxv0(ctx.dontUseScore());
-    engraving::layout::v0::ArpeggioLayout::layout(item, ctxv0);
+    engraving::layout::stable::LayoutContext ctxv0(ctx.dontUseScore());
```

Comment:
I think it should be just ctx (instead of ctxv0). Here and in other places too

Reply:
There is already a variable called `ctx`